server/src/main/java/org/elasticsearch/transport/TransportRequestDeduplicator.java

```diff
@@ -53,6 +53,14 @@ public void executeOnce(T request, ActionListener<Void> listener, BiConsumer<T,
         }
     }
 
+    /**
+     * Remove all tracked requests from this instance so that the first time {@link #executeOnce} is invoked with any request it triggers
+     * an actual request execution. Use this e.g. for requests to master that need to be sent again on master failover.
+     */
+    public void clear() {
```

Comment:
I know there's just one use case for this now, but just dropping all deduplication seems to be the right approach to master failovers in general if we ever want to  reuse this for e.g. put mapping calls or so.